cmd/mattermost/commands/channel.go

```diff
@@ -134,6 +145,7 @@ func init() {
 	ModifyChannelCmd.Flags().String("username", "", "Required. Username who changes the channel privacy.")
 
 	ChannelRenameCmd.Flags().String("display_name", "", "Channel Display Name")
+	SearchChannelCmd.Flags().Bool("team", false, "Team name or ID")
```

Comment:
This mustn't be a Bool, it should be a "String" with the team name or id. this way you can use any order, like `channel search myChannel --team myTeam`.